app/src/main/res/values/strings.xml

```diff
@@ -7,8 +7,8 @@
     <string name="local_storage">Local storage</string>
     <string name="favourites">Favourites</string>
     <string name="history">History</string>
-    <string name="error_occurred">An error has occurred</string>
-    <string name="network_error">Network connection error</string>
+    <string name="error_occurred">An error occurred</string>
+    <string name="network_error">Could not connect to the Internet</string>
```

Comment:
Not sure that it is correct

Reply:
https://en.wikipedia.org/wiki/Internet is correct.
Ultimately the problem could be just the local network, but if the objective is to always communicate externally, it isn't untrue.